platform/darwin/test/MGLFillStyleLayerTests.mm

```diff
@@ -137,6 +142,18 @@ - (void)testProperties {
                       @"Unsetting fillColor should return fill-color to the default value.");
         XCTAssertEqualObjects(layer.fillColor, defaultStyleValue,
                               @"fillColor should return the default value after being unset.");
+        // Transition property test
+        layer.fillColorTransition = transitionTest;
+        MGLTransition options;
+        auto toptions = rawLayer->getFillColorTransition();
+        options.delay = MGLTimeIntervalFromDuration(toptions.delay.value_or(mbgl::Duration::zero()));
+        options.duration = MGLTimeIntervalFromDuration(toptions.duration.value_or(mbgl::Duration::zero()));
+        XCTAssertEqual(options.delay, transitionTest.delay);
+        XCTAssertEqual(options.duration, transitionTest.duration);
```

Comment:
I would structure this a little differently:

* Set `fillColorTransition` to `transitionTest`.
* Call `getFillColorTransition()`.
* Assert `toptions.delay && *toptions.delay == transitionTest.delay`.